### ISSUE TITLE
Handle StringLists

### ISSUE THREAD

**Issue by chippyash** [mysense-ai/ServerlessPlugin-SSMPublish#67]
SSM Parameter store allows creation of StringLists for holding arrays of data:

Use case: securityGroupIds is an array of security group identifiers
Using configuration such as:
```
custom:
  vpc:
    securityGroupIds: ['sg-nnnnnnnnnnnnn','sg-mmmmmmmmmm']
  ssmPublish:
    enabled: true
    params:
      - path: /infrastructure/config/vpc/securityGroupIds
        value: ${self:custom.vpc.securityGroupIds}
        description: System VPC Security Group Ids
        secure: false
```
Will publish OK, but set the parameter type to 'String'
Retrieving it in another definition with : `${ssm:/infrastructure/config/vpc/securityGroupIds}` will generate an error such as
`An error occurred: SQSVPCEndpoint - Value of property SecurityGroupIds must be of type List of String`

Adding '~split' to the ssm call e.g.  `${ssm:/infrastructure/config/vpc/securityGroupIds~split}` results in error such as
`Cannot split SSM parameter '/infrastructure/config/vpc/securityGroupIds' of type 'String'. Must be 'StringList'`

I can see from the code that the plugin is trying to automatically determine the value content type and deal with it, but that only deals with objects, not arrays. I believe the plugin should also deal with arrays and convert them to string lists and set the parameter type correctly. At the moment it turns an array into a yaml array eg. `- value1\n - value2` whereas it should be a comma separated string e.g. `value1,value2`

**Comment by ColdFire87** [comment]
PR https://github.com/mysense-ai/ServerlessPlugin-SSMPublish/pull/68 adds support for explicitly specifying the type of param, eg `StringList`. Will merge it after the requested changes are pushed.

**Comment by monsagri** [comment]
This is addressed in PR #68, will be merged once requested changes are made.

**Comment by monsagri** [comment]
Resolved with #68 merged into master and released.
